InnerEye/ML/deep_learning_config.py

```diff
@@ -475,6 +475,7 @@ def logs_folder(self) -> Path:
     @property
     def checkpoint_folder(self) -> Path:
         """Gets the full path in which the model checkpoints should be stored during training."""
+        print(f"Expected Checkpoint path {self.outputs_folder / CHECKPOINT_FOLDER}")
```

Comment:
This print statement is very noisy, and clutters the output. We are using this property a LOT. Remove, or at least reduce it to rank zero only?
Also, I generally find it not ideal if properties have side effects.

Reply:
Removed. This was a leftover from debugging, thanks for spotting